sender/sender.cpp

```diff
@@ -0,0 +1,23 @@
+
+#include <iostream>
+#include "../sensor/temperature.h"
+#include "../sensor/state_of_charge.h"
+#include <vector>
+#include "sender.h"
+
+using namespace std;
+
+bool sendSensorDataToConsole(){
+  bool isDataSendSuccessfully = false;
```

Comment:
I think it require to further include negative case it the collection has empty data. Proabably same need to be check in Test cases